app/helpers/transcoding_status_helper.rb

```diff
@@ -0,0 +1,8 @@
+module TranscodingStatusHelper
+  def transcoding_status(model_instance)
+    meta = JSON.parse(model_instance.meta)
+    s = meta.fetch('job', {}).fetch('state', false)
+    "<div class=\"trans-status\">Transcoding status: " \
+      "#{s ? s : 'pending'}</div>\n"
+  end
+end
```

Comment:
On my local instance, I am getting the message "Transcoding status: pending" even when the transcoding has completed.  I will dig in and see if I can find a cause.